### DISCUSSION TITLE
how to handle bookmarked pages

### DISCUSSION THREAD

**Discussion by sainathdkpersonal** [facebook/docusaurus#7747] [Q&A] [answered]
Hi. This is my first attempt at using a staticsite generator and with docusaurus. I got the sample site working, built it and deployed to my test domain online. All the links are working but when I bookmark a child page/link or even open the link in a new window, I keep getting a 403. I am using GoDaddy windows based hosting. Following is a sample URL which when accessed directly returns a 403. However if I navigate from the left nav, I am able to get to the page with no issues. Any help/guidance is appreciated. Thanks!!

**trying below in browser returns 403**
`http://www.techwithme.net/dinodocsite1/docs/tutorial-extras/translate-your-site/`

**trying below brings up the sample site**
`http://www.techwithme.net/dinodocsite1/index.html`

**Comment by pranabdas** [comment, marked as answer]
I think it has something to do with your server configuration. Usually, when you visit a URL say <https://docusaurus.io>, the server shows the index file in that directory i.e., <https://docusaurus.io/index.html> **by default**. In your case, the server is not doing that. You can see that if you append `index.html` to your URLs (e.g., <http://www.techwithme.net/dinodocsite1/docs/tutorial-extras/translate-your-site/index.html>), it's working. By any chance, did you change or the default configuration is set to look for another default index file (say, `index.php`)? There should be option in your hosting settings.

**Reply by slorber** [reply]
definitively a hosting configuration issue: your provider returns 403 while the HTML find is found.
Configure it to allow serving the HTML file, this is outside the scope of Docusaurus and we can't really help.
I'd recommend using a modern static hosting provider like Vercel or Netlify

**Reply by sainathdkpersonal** [reply]
Thank you Pranabdas!!! I was initially playing around with .net webforms and MVC and razor. So I had deleted all the default documents except for default.aspx. I added back index.html which has resolved the problem. Yaay :)

**Reply by sainathdkpersonal** [reply]
> definitively a hosting configuration issue: your provider returns 403 while the HTML find is found. Configure it to allow serving the HTML file, this is outside the scope of Docusaurus and we can't really help. I'd recommend using a modern static hosting provider like Vercel or Netlify

Thanks Slorber for mentioning Vercel. They have a hobby pricing option which is the free tier. I have already paid for a year with GoDaddy that gave me around 8-9 USD per month. So will keep Vercel in mind on renewal.
